src/topology/instances/ennreal.lean

```diff
@@ -166,6 +166,14 @@ tendsto_nhds_top $ λ n, mem_at_top_sets.2
 by rw [tendsto_nhds_top_iff_nnreal, at_top_basis_Ioi.tendsto_right_iff];
   [simp, apply_instance, apply_instance]
 
+lemma tendsto_of_real_at_top : tendsto ennreal.of_real at_top (𝓝 ∞) :=
```

Comment:
This lemma is `tendsto_coe_nhds_top.2 (h : tendsto nnreal.of_real at_top at_top)`? It seems that `h` is missing. Could you add it, please?